src/fauxton/install.rst

```diff
@@ -96,6 +74,11 @@ layout, data, breadcrumbs and api point is required for the view.
 .. _backbone.js: http://backbonejs.org/
 .. _Backbone.layoutmanager: https://github.com/tbranyen/backbone.layoutmanager
 
+Writing Add-Ons
+---------------
+We have Fauxton Add-On documentation:
+    https://github.com/apache/couchdb-fauxton/blob/master/writing_addons.md
```

Comment:
These addon docs are so old. We need to update them before linking to them. Maybe we should leave this out